test/unit/random.pde

```diff
@@ -48,15 +48,15 @@ randomSeed(14);
 var minSample = 30, maxSample = 40;
 for(int i=0;i<attempts;++i) {
   var sample = r1[i];
-  if(sample < 0 || sample >= 1) _checkTrue(false); // bad random with no args
+  if (!(sample < 1 && sample >= 0)) _checkTrue(false); // bad random with no args
```

Comment:
hm, the old test code should have already done the right thing: if `sample` is less than 0 or it's greater or equal to one, it should throw an error,  by exploiting the fact that `false` will obviously fail a `checkTrue` check.

Can you explain how this rewrite improves on it? (it has certainly become harder to understand at a glance with this new ordering and effective double negative)